site/content/docs/5.2/getting-started/javascript.md

```diff
@@ -28,6 +28,11 @@ A better alternative for those using this type of frameworks is to use a framewo
 **Try it yourself!** Download the source code and working demo for using Bootstrap as an ES module from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/sass-js-esm). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/sass-js-esm?file=index.html).
```

Comment:
The javascript in stackblitz example isn't working for me (Bs side).

Reply:
What doesn't work in this example? Tried it and seems to work on Firefox/macOS.

Reply:
Can't reproduce it, so let's resolve this.